frontend/packages/helm-plugin/src/catalog/utils/catalog-utils.tsx

```diff
@@ -117,6 +117,11 @@ export const normalizeHelmCharts = (
             label: t('helm-plugin~Maintainers'),
             value: maintainers,
           },
+          {
+            label: t('helm-plugin~Documentation'),
+            value: '',
+            isDisabled: true,
+          },
```

Comment:
I am not 100% agree with this approach of adding a property to hide the property. Can we simply add a check if `documentationUrl`  is defined or undefined in `CatalogDetailsPanel` component?  It will hide the Documentation property Item for all types of catalog not only the helm and I think it is fine to hide it instead of showing N/A. wdyt? cc: @jerolimov

Reply:
Hi @vikram-raj ,

I had a discussion with @jerolimov , changed the property to `isHidden` and he is ok with this approach. 

I updated the PR. PTAL.